84.md

```diff
@@ -40,3 +40,10 @@ last value of the tag.
 ### Context
 Clients MAY include a `context` tag, useful when the highlight is a subset of a paragraph and displaying the
 surrounding content might be beneficial to give context to the highlight.
+
+## Quote Highlights
+A `comment` tag may be added to create a quote highlight. This should be rendered like a quote repost with the highlight as the quoted note.
+
+This is to prevent the creation and multiple notes (highlight + kind 1) for a single highlight action, which looks bad in micro-blogging clients where these notes may appear in succession.
+
+p-tag mentions should have a `mention` attribute to distinguish it from authors and editors.
```

Comment:
This looks good! In addition to this, I also had to add attributes for `r` tags as well in our implementation, to allow us to distinguish between a source highlight URL _(I named that attribute `"source"`)_ and a regular a URL attachment/mention _(I named it `"mention"`)_

Reply:
@jb55, should we add the above to the spec PR?

Reply:
just saw this, updating